FannyAndersson/PayWay
# Feature/change password

Hello!
User should always have an opportunity to change his or her password, even if he or she hasn't forgotten it yet. This commit is introducing new feature to change password via profile/settings.

How to test:
Log in.
Click menu -> Settings -> Change password
Fill a form
Click on Save. If success - you receive a success message. Click on button - you will be logged out and navigated to login page to log in with new password. 
Check mailbox - you receive a mail about password has been changed.
